<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VIDCON</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VIDCON</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filePathLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>113</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>파일 경로</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>428</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>115</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>불러오기</string>
    </property>
   </widget>
   <widget class="QLabel" name="horizontalLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Horizontal(H)</string>
    </property>
   </widget>
   <widget class="QLabel" name="verticalLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vertical(V)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="horizontalEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="verticalEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>215</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>215</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>px</string>
    </property>
   </widget>
   <widget class="QLabel" name="resolutionLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="addOptionCheckBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>312</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>고급 편집</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="editOptionLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stretch 옵션</string>
    </property>
   </widget>
   <widget class="QLabel" name="FPSLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>325</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="optionStripeRadio">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stripe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="optionZigzag1Radio">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zigzag(1by1)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="optionZigzag2Radio">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zigzag(2by1)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>384</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="fps60Radio">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>60FPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fps120Radio">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>120FPS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>영상 추출</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileFpsLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>216</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>216</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
   <widget class="QLabel" name="OptionsLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>기타 옵션</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="exceptOptionCheckBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>446</y>
      <width>251</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>원본 영상 제외</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
